include/fpu.h

```diff
@@ -28,6 +28,8 @@
 #include "mem.h"
 #endif
 
+#include "support.h"
```

Comment:
This include is no longer needed, I think. Probably should not be added in other headers any more either.

Reply:
Ah indeed! Thanks for catching these.